webapp/src/css/enketo/medic.less

```diff
@@ -18,10 +18,21 @@
     #form-languages {
       display: none;
     }
-    .option-wrapper {
+
+    .dropdown-menu .option-wrapper {
+      padding: 3px;
+      .option-label {
+        width: calc(100% - 34px);
+      }
+    }
+
+    .question:not(.or-appearance-likert) .option-wrapper {
+      padding: 5px;
       .option-label {
         white-space: normal;
-        word-break: normal;
+        word-break: break-all;
+        margin: 0;
+        display: contents;
```

Comment:
I suspect this is causing the bug that @tatilepizs found - translations probably have `display: none` and this is overwriting them.

Reply:
I commented on that line and all the translation go away, but this is how it looks:

![image](https://github.com/medic/cht-core/assets/94494491/a1e83f94-b6b4-42f8-82b5-78869d9d0dec)

Reply:
Fixed! Thank you!